tests/runtime/builtin

```diff
@@ -231,6 +231,6 @@ TIMEOUT 5
 NAME print args in kfunc
 PROG kfunc:vfs_open { print(args); exit(); }
 EXPECT { .path = 0x[0-9a-f]+, .file = 0x[0-9a-f]+ }
-REQUIRES_FEATURE btf
+REQUIRES_FEATURE kfunc
```

Comment:
AFAIK should be:
```suggestion
REQUIRES_FEATURE btf
REQUIRES_FEATURE kfunc
```

Reply:
in following commit we used same approach f2353019927351f0808befc5518c912e96c98f69

Reply:
Ok, let's keep consistent.